pdfrw/toreportlab.py

```diff
@@ -106,7 +106,8 @@ def _makearray(rldoc, pdfobj):
 
 def _makestr(rldoc, pdfobj):
     assert isinstance(pdfobj, (float, int, str)), repr(pdfobj)
-    return pdfobj
+    return str(getattr(pdfobj, 'encoded', pdfobj))
```

Comment:
I had a user-submitted issue related to floats -- it's not legal to use exponent notation on them.  Perhaps we should extract out the fix for them that I put into pdfwriter so that it can be used both there and here.